packages/frontend/src/components/Strategies/Crab/CrabTradeV2/Deposit.tsx

```diff
@@ -444,28 +468,22 @@ const CrabDeposit: React.FC<CrabDepositProps> = ({ onTxnConfirm }) => {
 
         <Box marginTop="24px">
           <Box display="flex" alignItems="center" justifyContent="space-between" gridGap="12px" flexWrap="wrap">
-            {!useQueue && (
-              <Metric
-                label="Uniswap Fee"
-                value={formatNumber(Number(uniswapFee)) + '%'}
-                isSmall
-                flexDirection="row"
-                justifyContent="space-between"
-                gridGap="8px"
-              />
-            )}
+            <Metric
+              label="Uniswap Fee"
+              value={useQueue ? '0 %' : formatNumber(Number(uniswapFee)) + '%'}
+              isSmall
+              flexDirection="row"
+              justifyContent="space-between"
+              gridGap="8px"
+            />
 
             <Box display="flex" alignItems="center" gridGap="6px" flex="1">
               <Metric
                 label={
                   <MetricLabel
                     label={useQueue ? 'Est. Price Impact' : 'Price Impact'}
                     tooltipTitle={
-                      useQueue
-                        ? `For standard deposit, the average price impact is ${formatNumber(
-                            depositPriceImpactNumber,
-                          )}% based on historical auctions`
-                        : undefined
+                      useQueue ? 'average price impact based on historical standard [deposits / withdraws]' : undefined
```

Comment:
ah what i meant here was have it say deposits if its on the deposit tab and withdraw if its on the deposit tab